frontend/src/App.tsx

```diff
@@ -252,13 +255,56 @@ function AppContainer() {
 
   React.useEffect(() => {
     if (desktopApi) {
-      desktopApi.receive(
-        'showReleaseNotes',
-        (data: { releaseNotes: string; appVersion: string }) => {
-          setReleaseNotes(data.releaseNotes);
-          setAppVersion(data.appVersion);
+      desktopApi.receive('appVersion', (currentBuildAppVersion: string) => {
+        setAppVersion(currentBuildAppVersion);
```

Comment:
Is the `appVersion` state actually needed here? I'd assume that since we store it in the localStorage an fetch it every time, there's no need to duplicate the state here. i.e. it can call `fetchRelease` with the `currentBuildAppVersion` and achieve the same AFAICS.

Reply:
done